frontend/.vscode/settings.json

```diff
@@ -86,7 +86,11 @@
   },
   "cucumberautocomplete.steps": ["./packages/*/integration-tests/support/step-definitions/*/*.ts"],
   "cucumberautocomplete.strictGherkinCompletion": true,
-  "editor.quickSuggestions": true
```

Comment:
Recent versions of VS code keep updating the settings file with this change.

Let me know if you think we should exclude this from the PR.